PowerEditor/installer/nppSetup.nsi

```diff
@@ -178,16 +178,28 @@ updaterDone:
 	Call SetRoughEstimation		; This is rough estimation of files present in function copyCommonFiles
 	InitPluginsDir			; Initializes the plug-ins dir ($PLUGINSDIR) if not already initialized.
 	Call checkCompatibility		; check unsupported OSes and CPUs
-		
+	
 	; look for previously selected language
 	ClearErrors
 	Var /GLOBAL tempLng
 	ReadRegStr $tempLng HKLM "SOFTWARE\${APPNAME}" 'InstallerLanguage'
 	${IfNot} ${Errors}
-		StrCpy $LANGUAGE "$tempLng" ; set default language
+		${If} "$tempLng" == "Installer Language" ; check for the possible previous inappropriate value (from some of the 'Afrikaans' issues...)

```

Comment:
PART2

So here could it be a function which detects whether if `$tempLang` is a legal value - all values other than legal values will fall down to  ${LANG_ENGLISH}?

Reply:
Yes.

I originally considered making my PR more universal here by taking advantage of the new `unsupportedLanguageToEnglish` function proposed by @mpheath (previous PR https://github.com/notepad-plus-plus/notepad-plus-plus/pull/14125). But then I thought that you would not like it because it increases the size of the PR (IMO - in the wild, there are unlikely any other invalid values than the NSIS generated "Installer Language" one.)

> The PR could be accepted as it is. However, a better management of exception could be added.
Please let me know if you want to work on it.

In fact, I welcome a more universal solution. I will extend my solution with the above mentioned `unsupportedLanguageToEnglish` function.